models/models.js

```diff
@@ -45,8 +45,12 @@ const selectAllArticles = (orderBy) => {
     })
   }
 
-const insertComment = (username, body, article_id) => {
-
+const insertComment = (newComment, article_id) => {
+    
+    if(Object.keys(newComment).length !== 2) {
```

Comment:
I do believe this section of error handling doesn't require a custom promise reject and could be handled by psql error handling functions.